### PR TITLE
improve: Add TestCase Verify CRUD Post

### PR DESCRIPTION
**Pull request by sieusaya149**
- Add testcase and fix code crud post

### CODE REVIEW

File: src/dbs/init.mysql.js [hunk 1]
```diff
@@ -55,7 +55,7 @@ class Database {
 
 	createNewConnection() {
 		return mysql.createConnection({
-			host: MYSQL_HOST || 'mysql',
+			host: MYSQL_HOST || 'mysql1',
```
Comment (hikijs) [inline src/dbs/init.mysql.js]: should revert this change

**Review by ghost — CHANGES_REQUESTED** [review]
Please revert the change in my comment

File: src/dbs/init.mysql.js (hunk 1 above)
Comment (sieusaya149) [reply src/dbs/init.mysql.js]: oke thanks
